src/command.rs

```diff
@@ -92,14 +117,16 @@ pub unsafe extern "C" fn wgpu_command_encoder_begin_render_pass(
 /// problems. For example, a double-free may occur if the function is called
 /// twice on the same raw pointer.
 #[no_mangle]
-pub unsafe extern "C" fn wgpu_render_pass_end_pass(pass_id: id::RenderPassId) {
-    let (pass_data, encoder_id) = Box::from_raw(pass_id).finish_render();
-    gfx_select!(encoder_id => GLOBAL.command_encoder_run_render_pass(encoder_id, &pass_data))
+pub unsafe extern "C" fn wgpu_render_pass_end_pass(pass: &mut RenderPass) {
```

Comment:
this is UB to receive `&mut RenderPass` here, because we are producing a box from it, and then we have two different objects uniquely pointing to the same memory...